source/SIL.AppBuilder.Portal.Frontend/src/lib/collection.ts

```diff
@@ -40,3 +40,10 @@ export function compareVia(accessor: any, invert?: boolean) {
     return 0;
   };
 }
+
+// This is a stupid way to 'deeply' compare things.
+// But it kinda works.
+// Functions are omitted from the comparison
+export function arePropsEqual(a, b) {
```

Comment:
perhaps a better name... because we could eventually use it to compare objects that are not props

Reply:
yeah, I just copy pasted this from the existing location